scripts/ingests/utils.py

```diff
@@ -103,15 +97,15 @@ def sort_sources(db, ingest_names, ingest_ras, ingest_decs, search_radius=60., v
     db_names = []
 
     for i, name in enumerate(ingest_names):
-        verboseprint("\n", i, ": searching:,", name, verbose=verbose)
+        logger.debug(''.join(map(str, ["\n", i, ": searching:,", name])))
```

Comment:
I recommend we switch these to f-strings, for example:
```
logger.debug(f"{i}: searching: {name}")
```
(I removed the \n since I don't think we really need that here)

This also applies to the other uses of join/map/str.
F-strings are a feature since python 3.6 and AstrodbKit2 has a minimum dependency of 3.7 or later so there is no reason for us not to use them.

Reply:
Oh yeah I totally agree and I love f strings normally; there were just 107 different print statements, most of which in the multiple args form; which was breaking logger (needed to be one single string); and I **really** didn't want to go through one by one to fix them... the `''.join(map(str, []))` method meant I could change every print statement at once thanks to pycharm...